oneview_redfish_toolkit/api/computer_system_collection.py

```diff
@@ -23,28 +23,30 @@ class ComputerSystemCollection(RedfishJsonValidator):
     """Creates a Computer System Collection Redfish dict
 
         Populates self.redfish with some hardcoded ComputerSystemCollection
-        values and with the response of Oneview with all server hardware.
+        values and with the response of Oneview with all servers profiled
+        applied.
```

Comment:
maybe it could be better to change a little bit to:"... with all servers with Server Profile applied"